org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/workspace/internal/SynchronizingBuildScriptUpdateListener.java

```diff
@@ -125,4 +132,19 @@ public static SynchronizingBuildScriptUpdateListener createAndRegister() {
     public void close() {
         ResourcesPlugin.getWorkspace().removeResourceChangeListener(this);
     }
+
+    /**
+     * Custom result handler that only logs the failure.
+     */
+    private static final class ResultHander implements ToolingApiJobResultHandler<Void> {
```

Comment:
⭕️ Why do we only log here? Before it would show an error dialog.

Reply:
This was intentional. If the `auto-sync` feature is enabled then a new error dialog would pop up every time the user saves a faulty build script file.